### PR TITLE
install

### PR DESCRIPTION
**Pull request by FernetMenta**

### CODE REVIEW

**Comment by FernetMenta** [comment]
@wsnipex mind having a look at this?

File: Makefile.in [hunk 1]
```diff
@@ -590,7 +582,8 @@ ifeq ($(findstring Darwin,$(shell uname -s)),Darwin)
 		-exec install "{}" $(DESTDIR)$(libdir)/xbmc/"{}" \; \
 		-exec printf " -- %-75.75s\r" "{}" \;
 else
-	@find system addons -regextype posix-extended -type f -not -iregex ".*\.git.*" -iregex ".*\.so|.*\.vis|.*\.xbs|.*\.pvr" -exec install -D "{}" $(DESTDIR)$(libdir)/@APP_NAME_LC@/"{}" \; -printf " -- %-75.75f\r"
+	@find system addons -regextype posix-extended -type f -not -iregex ".*\.git.*" -iregex ".*\.so|.*\.so\.[0-9].*|.*\.vis|.*\.xbs" -exec install -D "{}" $(DESTDIR)$(libdir)/@APP_NAME_LC@/"{}" \; -printf " -- %-75.75f\r"
+	@find -L addons -regextype posix-extended -type f -not -iregex ".*\.git.*" -iregex ".*\.so|.*\.so\.[0-9].*" -exec rsync -aRlq "{}" $(DESTDIR)$(libdir)/@APP_NAME_LC@/ \; -printf " -- %-75.75f\r"
```
Comment (wsnipex) [inline Makefile.in]: please use install instead of rsync. We don't check for/require rsync anywhere.

Comment (FernetMenta) [inline Makefile.in]: install does not copy symbolic links. but I can use cp -d. would this be ok?

Comment (wsnipex) [inline Makefile.in]: wouldn't it be enough to use install on the so? Otherwise cp is ok.

Comment (FernetMenta) [inline Makefile.in]: as already mentioned, install don't install symlinks which are required. i.e. pvr addons are loaded by searching for something like pvr.demo.so but the actual lib is named like  pvr.demo.so.12.1.0
pvr.demo.so -> pvr.demo.so.15.0
pvr.demo.so.15.0 -> pvr.demo.so.12.1.0

Comment (wsnipex) [inline Makefile.in]: well, install will follow the symlink and create a full pvr.demo.so, which can be loaded directly. Anyway, cp is ok for me as well.

**Pull request closed without merging**